net/service/handlers.go

```diff
@@ -330,7 +331,11 @@ func (s *TextileService) handleThreadAnnotation(pid peer.ID, pmes *pb.Envelope,
 		if err != nil {
 			return nil, err
 		}
-		notification.Body = "commented on " + target
+		body, err := thrd.Decrypt(annotation.CaptionCipher)
+		if err != nil {
+			return nil, err
+		}
+		notification.Body = fmt.Sprintf("commented on %s: \"%s\"", target, string(body))
```

Comment:
feels a bit like it would be nicer to have these split out so i can easily rearrange the text based on the type of display (app notification or feed row). fine for now